ARM-wvd-templates/nestedtemplates/managedDisks-customimagevm.json

```diff
@@ -305,6 +305,27 @@
         "description": "Session host configuration version of the host pool."
       },
       "defaultValue": ""
+    },
+    "securityType": {

```

Comment:
These parameters aren't passed to the virtual machine so it doesn't do anything.